tests/test_matplotlib.py

```diff
@@ -0,0 +1,260 @@
+#  Copyright 2019-2020 The Lux Authors.
```

Comment:
Why were these specific tests chosen? Can we add to `test_vis.py` and have one test for each chart type? Then run them through both Altair and Matplotlib for checks?